package.json

```diff
@@ -5,25 +5,22 @@
   "scripts": {
     "preinstall": "npx only-allow pnpm",
     "version": "pnpm i --frozen-lockfile=false && git add pnpm-lock.yaml",
-    "lerna": "lerna",
-    "bootstrap": "lerna bootstrap",
     "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
-    "prepack": "lerna run --stream prepack",
-    "dev": "lerna run --stream prepack -- --incremental && lerna --ignore=@logto/integration-tests run --parallel dev",
+    "prepack": "pnpm -r prepack",
+    "dev": "pnpm -r prepack --incremental && pnpm -r --parallel --filter=!@logto/integration-tests dev",
```

Comment:
`pnpm -r` doesn't seem to support `--parallel` config. The default concurrency is 4, and we can use `--workspace-concurrency=Infinite` for unlimited value
https://pnpm.io/cli/recursive#--workspace-concurrency